docs/integrations/ldap/org.md

```diff
@@ -314,3 +314,66 @@ builder.addProcessor(
   }),
 );
 ```
+
+## Using a Provider instead of a Processor
+
+An alternative to using the Processor for ingesting LDAP entries is to use a
+Provider. Doing this can give you a little bit more freedom to handle the LDAP
+ingestion more independently from the rest of the catalog ingestion.
+
+This can be useful if you have a lot of Users and Groups and hitting your LDAP
+server is resource intensive but you still want your other catalog entries to be
+updated frequently.
+
+> Note: When configuring to use a Provider instead of a Processor you do not
+> need to add a _location_ pointing to your LDAP server
+
+```ts
+// packages/backend/src/plugins/catalog.ts
+import { CatalogBuilder } from '@backstage/plugin-catalog-backend';
+import { PluginEnvironment } from '../types';
+
+import { Router } from 'express';
+import { LdapOrgEntityProvider } from '@backstage/plugin-catalog-backend-module-ldap';
+
+import { TaskScheduler } from '@backstage/backend-tasks';
+import { Duration } from 'luxon';
+
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  const scheduler = TaskScheduler.fromConfig(env.config).forPlugin('catalog');
```

Comment:
An updated backend [should already have](https://github.com/backstage/backstage/blob/master/packages/create-app/CHANGELOG.md#0410) a `scheduler` in the `env`, so no need to create a new one.

Reply:
Oh, cool 👍  We must've missed that update, fixed both on our end and in the docs now